src/demo-app/a11y/slider/slider-a11y.html

```diff
@@ -0,0 +1,36 @@
+<section>
+  <h2>Continuous slider (e.g. color component sliders)</h2>
```

Comment:
Add a `<p>` with something like `Select the red, blue, and green values for your color.` ?
(similar for other demos)